<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Low-pass filter Setup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Return to menu</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="PlotWidget" name="bode_diagram_amp"/>
    <widget class="PlotWidget" name="bode_diagram_phase"/>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Filter type</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="FIR_checkbox">
       <property name="text">
        <string>FIR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="Butterworth_checkbox">
       <property name="text">
        <string>Butterworth</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="Chebyshev1_checkbox">
       <property name="text">
        <string>Chebyshev1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="Elliptic_checkbox">
       <property name="text">
        <string>Elliptic (Cauer)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Filter order</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="num_coeff_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Cut-off frequency [Hz]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cut_off_fre_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Max. ripple [dB]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_ripple_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Min. attenuation [dB]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_attenuation_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>max_ripple_line</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>min_attenuation_line</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>max_ripple_line</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>min_attenuation_line</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Chebyshev1_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>min_attenuation_line</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Butterworth_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Chebyshev1_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Elliptic_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>FIR_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Chebyshev1_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Elliptic_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Chebyshev1_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>FIR_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Chebyshev1_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Butterworth_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Chebyshev1_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Elliptic_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Elliptic_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>FIR_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Elliptic_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Butterworth_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Elliptic_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Chebyshev1_checkbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FIR_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Butterworth_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Chebyshev1_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
